### PR TITLE
Add login middleware

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Allow to use playground with keycloak

## Current flow

1. localhost:8000/login
2. Login to keycloak
3. Copy token
4. Use token in the playground

http://www.giphy.com/gifs/ukgZfagWxYxkhqIQh2

## Investigation

Refresh playground tab with token after login (it's attached to req.session)

### CODE REVIEW

**Comment by wtrocki** [comment]
Ready for review!

File: server/security/index.js [hunk 1]
```diff
@@ -30,6 +30,13 @@ exports.applyAuthMiddleware = (keycloakConfig, expressRouter, apiPath) => {
     // Protect the main route for all graphql services
     // Disable unauthenticated access
     expressRouter.use(apiPath, keycloak.protect())
+
+    expressRouter.get('/login', keycloak.protect(), function (req, res) {
+      const token = JSON.stringify(JSON.parse(req.session['keycloak-token']))
+      res.send({
```
Comment (darahayes) [inline server/security/index.js]: If you change this to the following, it's a little bit handier.

```js
const token = JSON.parse(req.session['keycloak-token'])
res.json({ authorization: `Bearer ${token.access_token}`})
```

Comment (wtrocki) [reply server/security/index.js]: Yeah. done that already. Just haven't pushed :D

File: server/security/index.js [hunk 2]
```diff
@@ -25,11 +25,23 @@ exports.applyAuthMiddleware = (keycloakConfig, expressRouter, apiPath) => {
...
     // Disable unauthenticated access
     expressRouter.use(apiPath, keycloak.protect())
+
+    expressRouter.get('/login', keycloak.protect(), function (req, res) {
+      let token = req.session['keycloak-token']
+      if (token) {
+        token = JSON.parse(token).access_token
+      }
+      res.send({
+        'Authorization': 'Bearer ' + token
```
Comment (darahayes) [inline server/security/index.js]: Shouldn't this be token.access_token?

Comment (darahayes) [reply server/security/index.js]: Also `res.json` is better here

**Pull request merged**